src/store/store.ts

```diff
@@ -151,26 +151,36 @@ export const useStore = defineStore('store', {
       return (this.getCorrectedFilingType === FilingTypes.CHANGE_OF_REGISTRATION)
     },
 
-    /** The original entity type, this is for entity type alteration */
-    getOriginLegalType (): CorpTypeCd {
-      return this.getEntitySnapshot?.businessInfo?.legalType
+    /** The original business info. NB: may be null. */
+    getOriginalBusinessInfo (): BusinessInformationIF {
+      return this.getEntitySnapshot?.businessInfo || null
```

Comment:
Please verify that we use conditional chaining on `getOriginalBusinessInfo` since it might be null.

It would be OK if our fallback was `{}` (since we can check for properties on that) but don't change it.

Reply:
I did a global search Sev. All instances of `getOriginalBusinessInfo` use optional chaining (basically same as before).